packages/vats/src/core/chain-behaviors.js

```diff
@@ -382,6 +383,79 @@ export const makeChainStorage = async ({
   chainStorageP.resolve(rootNodeP);
 };
 
+/**
+ * @typedef HighPrioritySendersManager
+ * @property {(namespace: string, address: string) => Promise<void>} add
+ * @property {(namespace: string, address: string) => void} remove
+ */
+export const PRIORITY_SENDERS_NAMESPACE_RE = /^[a-zA-Z0-9_-]{1,50}$/;
+
+/**
+ * @param {BootstrapSpace & {
+ *   consume: { loadCriticalVat: ERef<VatLoader<ChainStorageVat>> }
+ * }} powers
+ */
+export const makeHighPrioritySendersManager = async ({
+  consume: { loadCriticalVat, bridgeManager: bridgeManagerP },
+  produce: { highPrioritySendersManager: managerP },
+}) => {
+  const bridgeManager = await bridgeManagerP;
+  if (!bridgeManager) {
+    console.warn('Cannot support chainStorage without an actual chain.');
+    managerP.resolve(null);
+    return;
+  }
+
+  const storageBridgeManager = E(bridgeManager).register(BRIDGE_ID.STORAGE);
+
+  /** @type {Map<string, StorageNode>} */
+  const addressNodes = new Map();
+
+  const vat = E(loadCriticalVat)('bridge');
+  const sendersNode = E(vat).makeBridgedChainStorageRoot(
+    storageBridgeManager,
+    STORAGE_PATH.HIGH_PRIORITY_SENDERS,
+    { sequence: true },
```

Comment:
Should be set not appended.

```suggestion
    { sequence: false },
```